src/components/button/button.tsx

```diff
@@ -30,12 +35,18 @@ const WhiteOnGreenButtonStyles = styled.button`
   height: 2rem;
   min-width: 13rem;
   border: none;
+  outline:none;
   border-radius: 0.2rem;
   background: ${props => props.theme.colors.pureWhite};
   color: ${props => props.theme.colors.greenDark};
-  font-size: 0.7rem;
+  font-size: ${props => props.theme.fontsize.paragraph};
   font-weight: bold;
   cursor: pointer;
+  width: 208px;
```

Comment:
Try not to put px but use rems, also don't hardcode the width until you are absolutely sure you would want this button to have a fixed width everywhere, else provide the width as props, would be more flexible.